packages/account/src/Sections/Security/AccountLimits/account-limits.jsx

```diff
@@ -105,137 +105,142 @@ class AccountLimits extends React.Component {
+                                                            'Represents the maximum volume of contracts that you may purchase in any given trading day.'
+                                                        )}
+                                                        margin={0}
+                                                    />
+                                                </div>
+                                            </TableHeader>
+                                            <TableHeader>{localize('Limit')}</TableHeader>
+                                        </Row>
+                                    </thead>
+                                    <tbody>
+                                        {commodities && makeTurnoverLimitRow(currency, commodities)}
+                                        {forex_ordered &&
+                                            makeTurnoverLimitRow(currency, forex_ordered, localize('Forex'))}
+                                        {indices && makeTurnoverLimitRow(currency, indices)}
+                                        {synthetic_index && makeTurnoverLimitRow(currency, synthetic_index)}
+                                    </tbody>
+                                </table>
+                                <table className='account-management-table'>
+                                    <thead>
+                                        <Row>
+                                            <TableHeader>{localize('Withdrawal limits')}</TableHeader>
+                                            {!is_fully_authenticated && <TableHeader>{localize('Limit')}</TableHeader>}
+                                        </Row>
+                                    </thead>
+                                    {!is_fully_authenticated && (
+                                        <tbody>
+                                            <Row>
+                                                <Td>{localize('Total withdrawal allowed')}</Td>
+                                                <Td>{formatMoney(currency, num_of_days_limit, true)}</Td>
+                                            </Row>
+                                            <Row>
+                                                <Td>{localize('Total withdrawn')}</Td>
+                                                <Td>
+                                                    {formatMoney(currency, withdrawal_since_inception_monetary, true)}
+                                                </Td>
+                                            </Row>
+                                            <Row>
+                                                <Td>{localize('Maximum withdrawal remaining')}</Td>
+                                                <Td>{formatMoney(currency, remainder, true)}</Td>
+                                            </Row>
+                                        </tbody>
```

Comment:
if `is_fully_authenticated` set to true, is this table going to render only a header? :thinking: is this by design?